src/source/feeds.rst

```diff
@@ -13,16 +13,67 @@ We treat the provided XML file as the desired list of ads the user wants to have
+:ref:`feed_budget`                     budget details                       :ref:`feed_budget`   no        
+:ref:`feed_ship`                       shipping options                     :ref:`feed_ship`     no
+:ref:`feed_phoneNumber`                phone number                         max. 32 chars        no        
+:ref:`feed_emailAdvertiser`            allow emails to the seller           true,false           no
+:ref:`feed_regionId`                   only applicable for Kijiji Canada    numeric              no        
+:ref:`feed_engines`                    targeted search engines for this ad  emunerated list      no
+:ref:`feed_microTip`                   tiny product hightlight              max. 18 chars        no
+:ref:`feed_mpn`                        Manufacturer Part Number (MPN)       len. 2-70 chars      no   
+:ref:`feed_googleProductCategory`      google category for your product     string               no
+:ref:`feed_productType`                customer product type                max. 750 chars       no    
+:ref:`feed_brand`                      product brand name                   max. 70 chars        no
+:ref:`feed_gtin`                       Global Trade Identification Number   max. 50 chars        no  
+:ref:`feed_itemGroupId`                groups product variants in your      len. 1-50 chars      no
+:ref:`feed_condition`                  condition of product                 enum                 no
+:ref:`feed_material`                   main product fabric or material      max. 200 chars       no
+:ref:`feed_energyEfficiencyClass`      energy efficiency class              enum                 no
+:ref:`feed_minEnergyEfficiencyClass`   minimal energy efficiency class      enum                 no
+:ref:`feed_maxEnergyEfficiencyClass`   maximal energy efficiency class      enum                 no
+:ref:`feed_color`                      product color                        len. 1-100 chars     no
+:ref:`feed_gender`                     gender product is designed for       enum                 no
+:ref:`feed_ageGroup`                   age group product is intended for    enum                 no
+:ref:`feed_size`                       size information                     enum                 no
+:ref:`feed_unitPricingBaseMeasure`     denominator for product unit price   string               no
+:ref:`feed_unitPricingMeasure`         measure and dimension of product     string               no
+====================================== ==================================== ===================  =========== 
+
+.. index:: vendorId
+.. _feed_vendorId:
+
+vendorId
+""""""""
+
 The (mandatory) **vendorId** field in the XML file is there to let us know, for consecutive imports, which
 ads are the same. This results in allowing us to update an existing ad with the same **vendorId** instead
 of creating a new ad. **vendorId** is mandatory and, as specified in the XSD, unique for each ad in the
```

Comment:
"Effectively also unique per seller", worth mentioning imho.

Reply:
it is there in line 75: " unique for each ad in the...."

Reply:
true, but I'd argue (minor) that the feed content can change (therefore the XML file can change), that's why is slightly more explicit to state that it implies on the seller level (we are going to check and match with existing, on the other hand, and we explain that). All good.